src/Apps/Order/Routes/Counter/index.tsx

```diff
@@ -0,0 +1,207 @@
+
+    return (
+      <>
+        <HorizontalPadding px={[0, 4]}>
+          <Row>
+            <Col>
+              <OrderStepper
+                currentStep="Review"
+                steps={counterofferFlowSteps}
+              />
+            </Col>
+          </Row>
+        </HorizontalPadding>
+
+        <HorizontalPadding>
+          <TwoColumnLayout
+            Content={
+              <Flex
+                flexDirection="column"
+                style={isCommittingMutation ? { pointerEvents: "none" } : {}}
+              >
+                <Flex flexDirection="column">
+                  <CountdownTimer
+                    action="Respond"
+                    note="Expired offers end the negotiation process permanently."
+                    countdownStart={order.lastOffer.createdAt}
+                    countdownEnd={order.stateExpiresAt}
+                  />
+                  <TransactionDetailsSummaryItem
+                    order={order}
+                    title="Your counteroffer"
+                    onChange={this.onChangeResponse}
+                    offerContextPrice="LAST_OFFER"
+                  />
+                </Flex>
+                <Spacer mb={3} />
+                <Flex flexDirection="column" />
+                <Media greaterThan="xs">
+                  <Button
+                    onClick={this.onSubmitButtonPressed}
+                    loading={isCommittingMutation}
+                    size="large"
+                    width="100%"
+                  >
+                    Submit
+                  </Button>
+                  <ConditionsOfSaleDisclaimer />
+                </Media>
+              </Flex>
+            }
+            Sidebar={
+              <Flex flexDirection="column">
+                <Flex flexDirection="column">
+                  <ArtworkSummaryItem order={order} />
+                  <ShippingSummaryItem order={order} locked />
+                  <CreditCardSummaryItem order={order} locked />
+                </Flex>
+                <Spacer mb={[2, 3]} />
+                <Media at="xs">
+                  <>
+                    <Button
+                      onClick={this.onSubmitButtonPressed}
+                      loading={isCommittingMutation}
+                      size="large"
+                      width="100%"
+                    >
+                      Submit
+                    </Button>
+                    <ConditionsOfSaleDisclaimer />
+                    <Spacer mb={[2, 3]} />
+                  </>
+                </Media>
+              </Flex>
+            }
+          />
+        </HorizontalPadding>
+
+        <ErrorModal
+          onClose={this.onCloseModal}
+          show={this.state.isErrorModalOpen}
+          contactEmail="orders@artsy.net"
+          detailText={this.state.errorModalMessage}
+          headerText={this.state.errorModalTitle}
+        />
+      </>
+    )
+  }
+}
+
+const ConditionsOfSaleDisclaimer = () => (
```

Comment:
Do you think it's worth extracting this into a separate component? Looks like we'll have to use it in each of the flows.